src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -131,11 +137,21 @@ class FlowHelper {
      */
     FlowPayload addFlow(FlowPayload flow) {
         log.debug("Adding flow '${flow.id}'")
+        cleanupManager.addAction({flowHelperV2.safeDeleteFlow(flow.getId())}, CleanupAfter.TEST)
```

Comment:
It seems like 'flowHelperV2.safeDeleteFlow' is not a safe way of flow deletion, and we can get an error during this action(maybe we need to skip waiting for a history deletion event, and just wait for flow absence). As a result, the rest of the auto-cleanup closures will be skipped.